examples/resteasy/src/main/java/io/smallrye/opentelemetry/example/resteasy/ExampleApplication.java

```diff
@@ -0,0 +1,40 @@
+package io.smallrye.opentelemetry.example.resteasy;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import javax.ws.rs.container.DynamicFeature;
+import javax.ws.rs.core.Application;
+
+import io.opentelemetry.exporters.logging.LoggingSpanExporter;
+import io.opentelemetry.exporters.otlp.OtlpGrpcSpanExporter;
+import io.opentelemetry.sdk.OpenTelemetrySdk;
+import io.opentelemetry.sdk.trace.TracerSdkProvider;
+import io.opentelemetry.sdk.trace.export.SimpleSpanProcessor;
+import io.smallrye.opentelemetry.jaxrs2.server.ServerTracingDynamicFeature;
+
+/**
+ * @author Pavol Loffay
+ */
+public class ExampleApplication extends Application {
+    private final HashSet<Object> singletons = new HashSet<>();
+
+    public ExampleApplication() {
+        TracerSdkProvider tracerProvider = OpenTelemetrySdk.getTracerProvider();
+        TracerSdkProvider.builder().build();
```

Comment:
Does the return value not need to be used in some way?

Seems odd to build something that doesn't appear to be stored anywhere inside those calls and then discard it

Reply:
This seems like a leftover. I will remove it.